src/test/Global.h

```diff
@@ -24,8 +24,8 @@
 #include <unordered_set>
 #include <vector>
 
-#include "Singleton.h"
 #include "TestObject.h"
+#include "utility/Singleton.h"
```

Comment:
```suggestion
#include "../utility/Singleton.h"
```

See https://github.com/DrCpp/DrMock/pull/1#discussion_r549805358